CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### Unreleased
   - Fix handling of None responses in remote call
-
+  - Changed status_code to status in cora_processor.py and request_helper.py for SDX logging
+  - Changed 'nack for retry' to 'nack' in consumer.py for SDX logging
```

Comment:
These generally also apply (apart from the blank line) to commit messages too (within reason - don't turn linguistic hoops just to shoehorn into a structure :smile: )